libcontainer/seccomp/patchbpf/enosys_linux.go

```diff
@@ -665,6 +665,9 @@ func filterFlags(config *configs.Seccomp, filter *libseccomp.ScmpFilter) (flags
 }
 
 func sysSeccompSetFilter(flags uint, filter []unix.SockFilter) (fd int, err error) {
+	// This debug output is validated in tests/integration/seccomp.bats
+	// by the SECCOMP_FILTER_FLAG_* test.
+	logrus.Debugf("seccomp filter flags: %d", flags)
```

Comment:
Any reason to not use `%v`? I find it more clear to use %v unless you want something else. Isn't %v printing the same here?

Reply:
So, `%d` says "print argument as an integer", and `%v` means "print argument using the default formatting for its type". Since the type is integer, in this case %v will result in the same output as %d. The only difference (I guess but not entirely sure), is %v incurs slightly more runtime overhead, and the only benefit of using %v here would be that we won't have to change it if we change the `flags` type.

Since this is clearly an integer, I don't see why we shouldn't use `%d`. Personally, I find it more clear, as I can see we're printing an integer without looking at the flags type.

So I have to ask the same question to you -- why not use %d?

Reply:
OK, it seems that you have ignored the description and reviewed the first two commits, which are actually from https://github.com/opencontainers/runc/pull/3580 and were merged in there.

Reply:
Oh, yes, I missed the description. Sorry!

I see reasons to one or the other, I personally prefer the other and was wondering if this was on purpose. SGTM if you prefer it :)